core/input/devices.js

```diff
@@ -154,7 +154,7 @@
         ['target',     'wo', 'dom'],  // DOM element that captures keyboard input
         ['focused',    'rw', 'bool'], // Capture and send key events
 
-        ['onKeyPress', 'rw', 'func'] // Handler for key press/release
+        ['onKeyEvent', 'rw', 'func'] // Handler for key press/release
```

Comment:
This is internal between the keyboard handler and the RFB object, so it should be fine.